### ISSUE TITLE
Disc Golf icons are bananas

### ISSUE THREAD

**Issue by johntyree** [open-austin/austingreenmap#26]
Zilker is pretty overwhelming. ![](https://cloud.githubusercontent.com/assets/108972/8643409/e660a20e-28f8-11e5-8440-2560291e1c7b.png)

Half of those aren't baskets. They are starting tees.

**Comment by luqmaan** [comment]
We use the AMENITY_TYPE field to come up with an icon.

Do we also need to look at the description?

**Comment by johntyree** [comment]
Looking at the description would be enough here and I can imagine it being useful in other contexts as well.

**Comment by luqmaan** [comment]
So how do you think we should fix this?

**Comment by johntyree** [comment]
I haven't had a change to dig through it, but it looks to me like we can just use some heuristics. If the description mentions basket, it's a basket, otherwise use a different icon (A frisbee? a circle? a square? Anything really, it will be more obvious and useful in context if someone were actually trying to use it.) Alternatively, if it mentions a tee we know it's a tee, otherwise it's a basket. Just something to differentiate the two cases.

**Comment by aaronmgdr** [comment]
maybe this would be of use https://github.com/bevacqua/fuzzysearch

**Comment by luqmaan** [comment]
Another solution is clustering the icons. But that will take some thinking to implement, since we use react-leaflet.

@johntyree can you link to some better icons from here? http://thenounproject.com

**Comment by luqmaan** [comment]
@johntyree look at the icons now

**Comment by johntyree** [comment]
Much better. Would be nice to be able to flip clustering off entirely (should be possible by tweaking zoom level) but this is better than it was, for sure.

**Comment by paulserraino** [comment]
@johntyree at max zoom we disabled clustering, because marker positions were misplaced and it looked kind of funky

https://github.com/open-austin/austingreenmap/blob/master/client/js/components/ParkMap.jsx#L110

**Comment by johntyree** [comment]
@paulserraino, right, but I'm saying that the problem can manifest before zooming that far, which is hard to detect, so just letting the user toggle it is an easy solution.
